<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>FMusic Lite</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Assets.qrc">
    <normaloff>:/Images/FMusic.ico</normaloff>:/Images/FMusic.ico</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">

#MainWindow {
    background-color: rgb(238,238,238);
}

/* QPush Button Styles */

QPushButton {
    background-color: #ffffff;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Microsoft YaHei UI&quot;;
}

QPushButton::hover {
    background-color: #F0F0F0;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Microsoft YaHei UI&quot;;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #ECECEC;
    border: 1px solid #ECECEC;
    border-radius: 5px;
    font: 9pt &quot;Microsoft YaHei UI&quot;;
}

/* QSlider Component Styles */
QSlider{
    border-color: #ffffff;
	border-top-color: rgb(255, 255, 255);
}

QSlider::groove:horizontal {                                
     border: 1px solid #999999;                             
     height: 1px;                                           
    margin: 0px 0;                                         
     left: 5px; right: 5px; 
}

QSlider::handle:horizontal {                               
     border: 0px;
     border-image: url(:/Images/handle_hover.png);
     width: 15px;                                    
     margin: -7px -7px -7px -7px;                  
} 

QSlider::add-page:horizontal{
background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #bcbcbc, stop:0.25 #bcbcbc, stop:0.5 #bcbcbc, stop:1 #bcbcbc); 

}

QSlider::sub-page:horizontal{                               
 background: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #439cf3, stop:0.25 #439cf3, stop:0.5 #439cf3, stop:1 #439cf3);                     
}

QSlider::handle:horizontal:hover {
     border: 0px;
     border-image: url(:/Images/handle_hover.png);
     width: 15px;                                           
     margin: -7px -7px -7px -7px;      
}

/* QLine Edit Styles */

QLineEdit {
	border: 1px solid #E3E3E3;
	border-radius: 5px;
	padding-left: 5px; 
	background-color: #FFFFFF;
	selection-background-color: #A0A0A0; 
	selection-color: #F2F2F2; 
    font: 9pt &quot;Microsoft YaHei UI&quot;;
}

QLineEdit:hover { 
	border: 1px solid #326CF3;
	border-radius: 5px;
	background-color: #FFFFFF;
	selection-background-color: #298DFF;
	selection-color: #F2F2F2;
}

QLineEdit[echoMode=&quot;2&quot;] { 
	lineedit-password-character: 9679;
	lineedit-password-mask-delay: 2000;
}

QLineEdit:disabled {
	border: 1px solid #CDCDCD;
	background-color: #CDCDCD;
	color: #B4B4B4;
}

QLineEdit:read-only {
	background-color: #CDCDCD;
	color: #F2F2F2;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title_bar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#title_bar {
   background-color: rgb(248,248,248);
   border-bottom: 1px solid #E0E0E0;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="home">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#home {
    background-color: rgba(0,0,0,0);
    border-radius: 0px;
    border: transparent;
    border-bottom: 2px solid #326CF3;
}</string>
        </property>
        <property name="text">
         <string>Home</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playlist">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#playlist {
    background-color: rgba(0,0,0,0);
    border-radius: 0px;
    border: transparent;
    border-bottom: 2px solid #E0E0E0;
}</string>
        </property>
        <property name="text">
         <string>PlayLists</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="me">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>75</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#me {
   
    background-color: rgba(0,0,0,0);
    border-radius: 0px;
    border: transparent;
    border-bottom: 2px solid #E0E0E0;
}</string>
        </property>
        <property name="text">
         <string>Me</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>25</height>
         </size>
        </property>
        <property name="inputMask">
         <string/>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Search Music...</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="shower" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#shower {

}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="controllerbar" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#controllerbar {
    
	background-color: rgb(255, 255, 255);
    border-top: 1px solid #E0E0E0;
    
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="TitleText">
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Microsoft YaHei UI&quot;;</string>
        </property>
        <property name="text">
         <string>TitleText</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="periousmusic">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #ffffff;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton::hover {
    background-color: #F0F0F0;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #ECECEC;
    border: 1px solid #ECECEC;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}
</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="playbutton">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe MDL2 Assets</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #ffffff;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton::hover {
    background-color: #F0F0F0;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #ECECEC;
    border: 1px solid #ECECEC;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QPushButton" name="nextmusic">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #ffffff;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton::hover {
    background-color: #F0F0F0;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #ECECEC;
    border: 1px solid #ECECEC;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="othermenu">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton {
    background-color: #ffffff;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton::hover {
    background-color: #F0F0F0;
    border: 1px solid #E3E3E3;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}

QPushButton:pressed, QPushButton:checked {
    background-color: #ECECEC;
    border: 1px solid #ECECEC;
    border-radius: 5px;
    font: 9pt &quot;Segoe MDL2 Assets&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Assets.qrc"/>
 </resources>
 <connections/>
</ui>
